tests/valid/string/multiline-empty-crlf.toml

```diff
@@ -0,0 +1,14 @@
+empty-1 = """"""
+
+# A newline immediately following the opening delimiter will be trimmed.
+empty-2 = """
+"""
+
+# \ at the end of line trims newlines as well; note that last \ is followed by
+# two spaces, which are ignored.
+empty-3 = """\
+    """
+empty-4 = """\
+   \
+   \  

```

Comment:
Given that we're encoding a raw `CRLF` into these files, we should also add a `.gitattributes` file with

```
*-crlf.toml    text eol=crlf
```

This can be useful for ensuring folks with `core.autocrlf=auto` get CRLF even on macOS/Linux.

Reply:
I think this already takes care of that? https://github.com/toml-lang/toml-test/blob/master/tests/.gitattributes

I think that should always stop it from frobbing with any newlines? Reading #48, that was the intent anyway.

Reply:
Ah, indeed!